scvi/module/_vae.py

```diff
@@ -112,6 +112,8 @@ def __init__(
         library_log_means: Optional[np.ndarray] = None,
         library_log_vars: Optional[np.ndarray] = None,
         var_activation: Optional[Callable] = None,
+        encoder_kwargs: dict = {},
+        decoder_kwargs: dict = {},
```

Comment:
not ideal to use mutable defaults, this should be `Optional[dict] = None`, also should be documented.

I would also considering calling these `extra_encoder_kwargs` because some init kwargs are passed to init of the encoder.

Reply:
@jjhong922 any thoughts on that?

Reply:
Yes agree with both your comments. I'm actually surprised codacy didnt complain about a mutable default!